packages/atomic/src/components/commerce/atomic-commerce-load-more-products/e2e/atomic-commerce-load-more-products.e2e.ts

```diff
@@ -0,0 +1,63 @@
+import {test, expect} from './fixture';
+import {AtomicCommerceLoadMoreProductsLocators} from './page-object';
+
+test.beforeEach(async ({page}) => {
+  await page.goto(
+    'http://localhost:4400/iframe.html?id=atomic-commerce-load-more-products--in-page&viewMode=story'
+  );
+});
+
+test('should be A11y compliant', async ({loadMore, makeAxeBuilder}) => {
+  await loadMore.hydrated.waitFor();
+  const accessibilityResults = await makeAxeBuilder().analyze();
+  expect(accessibilityResults.violations).toEqual([]);
+});
+
+test('should display a recap of the amount of results', async ({loadMore}) => {
+  await await expect(loadMore.summary()).toBeVisible();
+});
+
+test('should display a load more button', async ({loadMore}) => {
+  await loadMore.button.waitFor({state: 'visible'});
+  await expect(loadMore.button).toBeVisible();
+});
+
+test('should display a progress bar', async ({loadMore}) => {
+  await expect(loadMore.progressBar).toBeVisible();
+});
+
+test('should a progress value between 1 and 100', async ({loadMore}) => {
+  await expect(loadMore.progressValue).toHaveCSS('width', /^.+$/);
+});
+
+test.describe('after clicking the load more button', () => {
+  const getTotal = async (loadMore: AtomicCommerceLoadMoreProductsLocators) => {
+    const recap = await loadMore.summary().textContent();
+    const totalMatch = recap?.match(/\bshowing ([0-9]+) of/i);
+    return totalMatch ? parseInt(totalMatch[1]) : 0;
+  };
+
+  test('should load more products', async ({loadMore, products, page}) => {
+    await products.hydrated.waitFor();
+    const initialProductCount = await products.results.count();
+    await loadMore.button.click();
+    await page.waitForTimeout(1000);
+
+    const finalProductCount = await products.results.count();
+    const summaryTotal = await getTotal(loadMore);
+
+    expect(finalProductCount).toBeGreaterThan(initialProductCount);
+    expect(summaryTotal).toEqual(finalProductCount);
+  });
+});
```

Comment:
The product count at any point should be deterministic (we won't have too few products), so numeric comparison should be avoided.
Instead, create a selector for the loadMoreProduct text a bit like:
https://github.com/coveo/ui-kit/blob/087f507664e4fd3e589c5cadf5940df45c205a80/packages/atomic/src/components/commerce/atomic-commerce-search-box/e2e/page-object.ts#L17-L27
and use it for your assertions